src/main/java/net/snowflake/client/jdbc/telemetry/TelemetryClient.java

```diff
@@ -191,6 +191,21 @@ public static Telemetry createTelemetry(SFSession session, int flushSize) {
    * @param serverUrl server url
    * @return a telemetry connector
    */
+  public static Telemetry createSessionlessTelemetry(
+      CloseableHttpClient httpClient, String serverUrl) {
+    // By default, use KEYPAIR_JWT as the auth type
```

Comment:
> // By default, use KEYPAIR_JWT as the auth type

The users deserve to see that information in the method's javadoc comment, I would say.

Reply:
Updated